### DISCUSSION TITLE
Tree: scroll to node

### DISCUSSION THREAD

**Discussion by jasdom** [iTwin/iTwinUI#2095] [Ideas]
Hi, I am implementing a search box that allows stepping through search results by focusing on the next node that matches the search input, however when a virtualized tree is used some of the nodes might not be rendered and cannot be scrolled to. Is there a way this can be solved? If not, maybe support for a way to scroll to a node could be added?

**Comment by mayank99** [comment]
This is not currently supported, but there is an open issue for it: https://github.com/iTwin/iTwinUI/issues/902

We are currently reworking Tree virtualization. Once that is done, we can look into exposing "scroll to node" functionality. /cc @Ben-Pusey-Bentley
